feat(devtools): add searchbar

ferrettin opened the pull request:
#57 
![Parrot Devtools](https://user-images.githubusercontent.com/8594068/87428262-52eda780-c59f-11ea-8dcf-253dc8842306.png)

infoxicator commented on packages/parrot-devtools/__tests__/app/hooks/useScenarios.spec.js:
```diff
@@ -137,6 +139,15 @@ describe('useScenarios', () => {
     expect(global.fetch).toHaveBeenCalledTimes(2);
   });
 
+  it('should filter scenarios when filterValue is changed', () => {
```
We probably need another happy path test that filters out and returns just the filtered values in the array rather than empty

nellyk commented on packages/parrot-devtools/__tests__/app/components/ScenariosDisplay.spec.jsx:
```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 American Express Travel Related Services Company, Inc.
```
yes, this need to be updated

ferrettin commented:
Hey @nellyk and @infoxicator, thanks for taking a look at my PR. I've addressed the comments above, please let me know if any other changes need to be made
